CHANGELOG.md

```diff
@@ -19,18 +19,46 @@
 
 # Table of Contents
 
+- [2.2.0](#220)
 - [2.1.1](#211)
 - [2.1.0](#210)
 - [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
 
+# 2.2.0
```

Comment:
Should we use `2.2` here?

Reply:
ok for me, cc @nic-chen

Reply:
I think this is OK now, we have been using it like this.
https://raw.githubusercontent.com/apache/apisix/master/CHANGELOG.md